app/assets/scripts/views/countries.js

```diff
@@ -613,12 +613,12 @@ class AdminArea extends SFPComponent {
                     {data.key_priorities ? <ReactMarkdown source={data.key_priorities} /> : null}
                   </Fold>
                 </TabContent>
-                <TabContent isError={!get(this.props.keyFigures, 'data.results.length')} errorMessage={ NO_DATA } title="Key Figures">
+                <TabContent showError={true} isError={!get(this.props.keyFigures, 'data.results.length')} errorMessage={ NO_DATA } title="Key Figures">
```

Comment:
I added some comments to my changes in `tab-component.js` -- we are changing the default situation to be that a box is not shown at all if it has an error - i.e. it has no data. So now if there `isError`, then by default, that TabContent just renders as `null`.

There are some cases in which we do want to show an error, therefore TabContent now accepts an additional prop called `showError`, which, if set to true, will show the error panel instead of rendering `null`.

Maybe that prop can be called better, `showError` sounds misleading? What its representing is - "Show the error panel if there is an error", since the default is now "render `null` if there is an error".

Hopefully the comments in the TabContent make it a little clearer.

Thanks for the feedback here.